CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ Full documentation for rocDecode is available at [https://rocm.docs.amd.com/proj
 
 * Fixed a bug in picture files sample "videoDecodePicFiles" that can results in incorrect output frame count.
 * Fixed a decoded frame output issue in video size change cases.
+* Removed incorrect asserts of bitdepth_minus_8 in GetBitDepth() and num_chroma_planes in GetNumChromaPlanes() API calls in RocVideoDecoder utility class.
```

Comment:
I see that many apis are changed to the one without assert. So may be change the comment accordingly instead of listing just 2 cases